clamav-http/server/v0/scan_reply_handler.go

```diff
@@ -0,0 +1,56 @@
+package v0
+
+import (
+	"net/http"
+
+	"github.com/dutchcoders/go-clamd"
+	"github.com/sirupsen/logrus"
+)
+
+type ScanReplyHandler struct {
+	Address      string
+	Max_file_mem int64
+	Logger       *logrus.Logger
+}
+
+func (srh *ScanReplyHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+	err := r.ParseMultipartForm(srh.Max_file_mem * 1024 * 1024)
+
+	if err != nil {
+		w.WriteHeader(http.StatusInternalServerError)
+		w.Write([]byte("not okay"))
+		return
+	}
+
+	files := r.MultipartForm.File["file"]
+
+	if len(files) == 0 {
+		w.WriteHeader(http.StatusOK)
```

Comment:
As with the v1alpha scan handler, I think this fits a little better.
```suggestion
		w.WriteHeader(http.StatusNoContent)
```

Reply:
Agreed but leaving this as is for backwards compatbility